### ISSUE TITLE
How to use this plugin?

### ISSUE THREAD

**Issue by Waterbyte** [marytts/gradle-marytts-voicebuilding-plugin#101]
I couldn't find any documentation regarding the usage of this plugin. I am using marytts via server-client model. If I need to make a new voice, the old method of voice import doesn't work and this plugin is recommended. However, there is no way described to use this. Can you give me some general directions regarding its usage?

**Comment by psibre** [comment]
The plugin is under active development, but an older version is used for all of our voices (the marytts/voice-* repositories, e.g., [marytts/voice-cmu-slt](https://github.com/marytts/voice-cmu-slt)). Please have a look there.

Documentation will follow in the near future... =)

**Comment by Waterbyte** [comment]
Thanks for the reply. Sorry for bothering you but I looked at the repository. As far as I inferred, I downloaded this gradle-marytts-voicebuilding-plugin using git clone method. cd into the plugin folder. Installed software required for the plugin to work like sox, speech-tools, etc and from the root of the plugin folder, I ran the following command: 
`./gradlew legacyInit`.   The command didn't work as there was no task called legacyInit. Did I proceed the correct way?
